systemd/ceph-exporter@.service.in

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Ceph cluster exporter daemon
+PartOf=ceph-exporter.target
+After=network-online.target local-fs.target time-sync.target
+Before=remote-fs-pre.target ceph-exporter.target
+Wants=network-online.target local-fs.target time-sync.target remote-fs-pre.target ceph-exporter.target
+
+[Service]
+Environment=CLUSTER=ceph
+EnvironmentFile=-@SYSTEMD_ENV_FILE@
+ExecReload=/bin/kill -HUP $MAINPID
+ExecStart=/usr/bin/ceph-exporter -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
+LimitNOFILE=1048576
+LimitNPROC=1048576
```

Comment:
i am not sure if ceph-exporter requires these two settings, as it just an HTTP server, which write/read the socket files. and the number of connected HTTP clients and the number of opened socket files would be very small, so i guess the default `ulimit -n` and `ulimit -u` would suffice.

Reply:
Thanks. I removed these two settings.